xml/dhcp_vendor_class.xml

```diff
@@ -399,4 +399,4 @@
     <param pos="0" name="os.certainty" value="0.1"/>
   </fingerprint>
 
-</fingerprints>
+</fingerprints>
```

Comment:
The "no newline at end of file" is probably the dumbest thing Github complains about.

Reply:
It's helpful since you might not be aware. In many cases it is a non-issue so I understand your sentiment.